src/python/pants/backend/sql/lint/sqlfluff/rules.py

```diff
@@ -61,22 +65,15 @@ class _RunSqlfluffRequest:
     mode: SqlfluffMode
 
 
-@rule(level=LogLevel.DEBUG)
 async def run_sqlfluff(
```

Comment:
Did you do this manually, or did the migration tool detect that this didn't need to be a rule any more?

Reply:
I don't think it should have ever been a rule. After call-by-name, I was getting errors complaining about unhashable types. After looking at ruff, cue, and a few other backends - there is a common pattern of using this as a rule_helper (basically), rather than a rule itself.

Reply:
Good catch. Rule helpers previously had the `@rule_helper` decorator, but has since been superseded by just being any async method without being decorated, so this was an oversight on our part--should never have had the `@rule` decorator.

Reply:
The crazy part was that it worked when in `Get` syntax 🤯 No unhashable types problems